README.md

```diff
@@ -20,16 +20,51 @@ Once the binary file has been obtained, users will need to update their environm
 
 Example for Ubuntu:
 ```
-export PATH={BANDAGE_DIR}:$PATH
+export PATH=~/projects/Bandage:$PATH
 ```
 
 Example for MacOS:
 ```
-export PATH={BANDAGE_DIR}/Bandage.app/Contents/MacOS:$PATH
+export PATH=~/projects/Bandage/Bandage.app/Contents/MacOS:$PATH
```

Comment:
As I was installing YEAT on my WSL2 machine and following the README, I added the curly braces for Bandage into my PATH. It took me a few minutes to figure out what I did wrong. I wouldn't be surprised if others will make the same mistake as me. Maybe showing an example path would be more beneficial than "copying-and-replacing" the `{BANDAGE_DIR}`?